run-integrated-tiger.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+set -ex
+
+# cd "$(dirname "$0")/.."
+
+log() {
+  local now;
+  now=$(date '+%Y-%m-%d %H:%M:%S')
+  echo "$now $*" 1>&2
+}
+
+# docker network create aio-net
+
+docker run --name gooddata-cn-ce-manual -e APP_LOGLEVEL=INFO -e LICENSE_AND_PRIVACY_POLICY_ACCEPTED=YES \
```

Comment:
Container name should be unique -> Add `${EXECUTOR_NUMBER}` 
And should be a variables ?

Reply:
Same for `network`, it should be created with unique name + put into `trap` to be removed later.

`trap`: see comments below